webhook.go

```diff
@@ -66,9 +66,17 @@ func (m *SecretHubMutator) Mutate(ctx context.Context, obj metav1.Object) (bool,
 		containers[container] = struct{}{}
 	}
 
-	version, ok := pod.Annotations["secrethub.io/version"]
-	if !ok {
-		version = "latest"
+	image := ""
+	imageOverride, ok := pod.Annotations["secrethub.io/imageOverride"]
```

Comment:
I'm afraid there might be some confusion about whether you can/should use `secrethub.io/imageOverride` and `secrethub.io/version` annotations together or should supply the full `<image>:<tag>` format to this `imageOverride`.

How would you feel about: `secrethub.io/image/name` and `secrethub.io/image/tag` as an alternative? (we'll keep `secrethub.io/version` as alias to `secrethub.io/image/tag`, also for backwards compatibility)

As an additional benefit, I think `secrethub.io/image/tag` also makes it clearer that the tag format `1.0.0` should be used and not the version format `v1.0.0`

Reply:
And if you want an option to provide both in the same annotation, maybe we can also add `secrethub.io/image`

Reply:
Simon's proposal sounds good to me.

Only change I would suggest is to stick to conventional names for annotations, which is kebab-case and no `/` (cannot find any examples where that is used).

So that would mean:
```
secrethub.io/image-name
secrethub.io/image-tag
secrethub.io/version
```